WindowsServerDocs/remote/remote-access/directaccess/single-server-wizard/da-basic-plan-s1-infrastructure.md

```diff
@@ -110,6 +110,17 @@ In a DirectAccess deployment, DNS is required for the following:
 
     2.  If the corporate network is IPv6-based, the default address is the IPv6 address of DNS servers in the corporate network.
 
+> [!NOTE]
+> Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy (NRPT).
+> If the DNS registration is needed, for e.g. Manage Out, it can be explicitely enabled with this registry key on the client:
```

Comment:
Please note: use either A) "for example" or only "e.g." (exempli gratia, meaning "for example")
```suggestion
> If DNS registration is needed, for example by **Manage Out**, it can be explicitly enabled with this registry key on the client:
```
Also: typo correction, and add emphasis to the key phrase Manage Out.